src/Admin/AdminHelper.php

```diff
@@ -25,6 +25,8 @@
 use Symfony\Component\Form\FormView;
 
 /**
+ * @final since sonata-project/admin-bundle 3.0
```

Comment:
`3.0` is not really accurate, is it?

Reply:
No, they should be `3.x`. I'll try to update these comments soon. Thank you!

Reply:
Sorry for bearing the bad news :sweat_smile:

Reply:
I'm glad to have news from you :smile: 

These annotations should be fine now.